### PR TITLE
Adding composer.json

### PR DESCRIPTION
**Pull request by leek**
Simply adding a `composer.json` file so this plugin can be installed via Composer.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "boonebgorges/wordpress-importer",
```
Comment (GaryJones) [inline composer.json]: The owner needs updating.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "WordPress Importer",
+    "type": "wordpress-plugin",
+    "license": "GPLv2+",
```
Comment (GaryJones) [inline composer.json]: The license should now be `"GPLv2-or-later"`

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "keywords": [
+        "wordpress",
+        "plugin"
+    ],
+    "require": {
+        "php": ">=5.3.0",
```
Comment (GaryJones) [inline composer.json]: While Composer needs 5.3, what code in this plugin makes it 5.3?

**Comment by ocean90** [comment]
Closing for now. A proper file should be added as part of #30.

**Pull request closed without merging**